compute/src/test/java/org/jclouds/compute/callables/RunScriptOnNodeUsingSshTest.java

```diff
@@ -103,7 +103,7 @@ public void simpleRootTestWithSudoPassword() {
       expect(sshClient.getUsername()).andReturn("tester");
       expect(sshClient.getHostAddress()).andReturn("somewhere.example.com");
       expect(
-            sshClient.exec("sudo -S sh <<'RUN_SCRIPT_AS_ROOT_SSH'\n" + "testpassword!\n" + "echo $USER\n"
+            sshClient.exec("sudo -S sh <<'RUN_SCRIPT_AS_ROOT_SSH'\n" + "'testpassword!'\n" + "echo $USER\n"
```

Comment:
Would it make sense to modify this test so that it actually reflects the problem we're trying to fix (e.g. space in the password)? Also, would it be worth adding a test for the additional code path (see comment above)?

Reply:
+1. Let's change the default value used in tests to a conflicting one. @danielestevez can you have a look at these comments and submit a PR?

Reply:
@demobox 
That line of code https://github.com/jclouds/jclouds/pull/1205#discussion-diff-187774732R118 is already covered in simpleRootTest and testUserAddAsRoot tests

This PR adds a conflictive password for all tests https://github.com/jclouds/jclouds/pull/1207